### PR TITLE
Feature: Improving architecture using bloc

### PR DESCRIPTION
**Pull request by devdave25**
I have created a Bible Bloc, this is a layer of abstraction. This slots in-between the UI layer and the service/data layer. This is useful to have 1 source (e.g. the state) rather than having to call the services to get fresh data.

bloc_architecture_full

```
          // This is an example how to use the bloc
          BlocBuilder<BibleBloc, BibleState>(
            builder: (context, state) {
              if (state is BibleLoaded) {
                return const Text('LOADED');
              }
              return const Text('LOADING');
            },
          ),
```

### CODE REVIEW

**Comment by cr1pto** [comment]
You are a mind reader.  Bloc was one of the things I wanted to add in. :D

File: README.md [hunk 1]
```diff
@@ -42,3 +42,23 @@ ruby 2.6.8p205 (2021-07-07 revision 67951) [universal.arm64e-darwin21]
 - Run `flutter doctor` to ensure that everything works as expected from the command line.
 ### Running the app
 
+
+### Running the app wireless
```
Comment (cr1pto) [inline README.md:46]: This section is not necessary with Android 11+.  Android studio supports Wireless Debugging. Maybe we include that above this and keep what you currently have if someone doesn't want to use wireless debugging... thoughts?

**Review by cr1pto — COMMENTED** [review]
This LGTM; however, I'd like to run some manual tests to ensure I understand this PR.  I am very excited for these changes because Bloc was on my roadmap.  Thanks so much @devdave25 !

File: README.md (hunk 1 above)
Comment (devdave25) [reply README.md:46]: Maybe you could show me how to do that hehe, I was unable to work it.

**Review by cr1pto — APPROVED** [review]

**Pull request merged**